BWAPI4J/src/main/java/org/openbw/bwapi4j/unit/Addon.java

```diff
@@ -1,36 +1,7 @@
-////////////////////////////////////////////////////////////////////////////////
-//
-//    Copyright (C) 2017-2018 OpenBW Team

```

Comment:
Copyright notices should be in all Java and C++ files regardless of their contents.